application.yml

```diff
@@ -26,6 +26,8 @@ webtoken:
   timeout:
     in:
       minutes: 15
+jwt:
+  secretKey: ${JWT_SECRETKEY:XXXX}
```

Comment:
Should we provide a default value of `XXXX` for all secrets? What happens if the default value is actually used? I assume the application should not work with `XXXX`, or even if it does we do not actually want to use `XXXX`? Is any error thrown if we try to use `XXXX`?

Reply:
Removed the default value of `xxxx`.